content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,573 @@
+ 
+### T-Test with Python implementation
+The T-test is utilized when we plan to evaluate the discrepancy between the means of two groups of samples. Unlike the Chi-Square Test, the T-test is used on continuous variables.
+ 
+T-tests has three basic types
+ 
+1. **One sample t-test:** It checks whether the population has a different mean from the sample.
+The mathematical expression is:
+
+[one-sample-t-test](/engineering-education/statistical-hypothesis-testing-python-implementation/one-sample-t-test.PNG)
+ 
+where,
+the observed mean group is a,
+ 
+the assumed population means is B,
+ 
+the standard deviation of the data group is X,
+ 
+the number of observations in the group is N.
+ 
+2. **Two-sample t-test:** It checks and compares the means of two groups that are not dependent on each other and compares the population's means to see if there is a huge difference.
+The mathematical expression is:
+
+[two-sample-t-test](/engineering-education/statistical-hypothesis-testing-python-implementation/two-sample-t-test.PNG)
+
+where,
+data sample A's mean is Ma,
+
+data sample B's mean is Mb,
+ 
+the size of sample A is Na,
+ 
+the size of sample B is Nb,
+ 
+variance is v.
+ 
+3. **Paired t-test:** It is used in checking and comparing the means of different samples from a group.
+Mathematically, it can be resolved with:
+ 
+[paired-t-test](/engineering-education/statistical-hypothesis-testing-python-implementation/paired-t-test.PNG)
+ 
+where,
+the difference between paired observations' sample mean is D,
+ 
+the assumed mean difference is m,
+ 
+standard deviation is xd,
+ 
+the number of observations in the group is V.
+ 
+#### Performing a One-Sample T-Test
+```python
+df_ = df.petal_width
+```
+```python
+print(df_.sample(5))
+```
+```bash
+103    1.8
+131    2.0
+114    2.4
+35     0.2
+38     0.2
+Name: petal_width, dtype: float64
+```
+```python
+print(stats.ttest_1samp(a=df_, popmean=1.199))
+```
+```bash
+Ttest_1sampResult(statistic=-0.0053494404016899925, pvalue=0.9957389399651244)
+```
+**H0:** The mean of petal_width is 1.199
+ 
+**H1:** The mean of petal_width is not 1.199
+ 
+Analyzing one-sample t-test, it finds out if the hypothesized mean is the same or different from the group's mean. From the example above, we've selected the petal_width as the population sample to perform the test on.
+From the test, since the P-value beats the alpha level set at 0.05, we accept the null hypothesis because we don't have enough evidence to prove otherwise, moreover, if you check, the P-value is very close to 1.0, this means that the mean of petal_width is most likely 1.199.
+The evidence is too strong to be rejected, it is almost accurate. (Obviously, I copied the value of mean from the summary).
```

Comment:
The second paragraph can be put in two statements, and do add whitelines between paragraphs.
And please remove the sentence between brackets in the last paragraph.